madgiraffestudios/src/index.js

```diff
@@ -1,9 +1,29 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
 import './index.css';
 import App from './App';
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
- [ ] don't think this belong here or is the proper way to import bootstrap, please check
- [ ]  move bootstrap import into app.scss - should look similar to this: 
@import "~bootstrap/scss/bootstrap"; If it's already in app.scss, then just delete this import here